pkg/task/v1alpha1/runtask_command.go

```diff
@@ -43,17 +50,21 @@ const (
 type RunCommandCategory string
 
 const (
-	JivaCommandCategory   RunCommandCategory = "jiva"
-	CstorCommandCategory  RunCommandCategory = "cstor"
+	// JivaCommandCategory represents jiva command category
```

Comment:
`JivaCommandCategory categorises the run command as cstor based`

Can you please rephrase all the comments accordingly.